### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by 0xdewy** [liquality/chainify#150]
I'm getting the following error when trying to run the example script. 

```
/btc20node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js:1
ReferenceError: regeneratorRuntime is not defined
    at /btc20/node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js:86:50
    at /btc20/node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js:125:6
    at Object.<anonymous> (/btc20/node_modules/@ledgerhq/hw-transport-node-hid/lib/TransportNodeHid.js:202:2)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
```

**Comment by harshjv** [comment]
Hey @kyledewy. Thank you for trying `chainanstractionlayer`!

This is because one of the dependencies is using old version of babel. We are tracking it here: https://github.com/LedgerHQ/ledgerjs/issues/211 & https://github.com/LedgerHQ/ledgerjs/issues/218

We have fixed this issue in the `next` release of `chainabstractionlayer`. You can install it using `npm i @liquality/chainabstractionlayer@next`.

Or, you can do `npm install regenerator-runtime` locally in your project.

Cheers!

**Comment by harshjv** [comment]
We've just published `0.1.21` on npm with the fix. (`npm i @liquality/chainabstractionlayer`)
